Unix/installbuilder/conf/omiserver.conf

```diff
@@ -16,9 +16,11 @@ httpsport=0
 #idletimeout=TIMEOUT
 
 ##
-## trace -- enable tracing to standard output (default is 'false')
+## loglevel -- set the loggiing options for MI server
+##   Valid options are: ERROR, WARNING, INFO, DEBUG, VERBOSE (debug builds only)
+##   If commented out, then default value is: WARNING
 ##
-#trace=(true|false)
+#loglevel = INFO
```

Comment:
Generally, when we comment out, uncommenting gets you the default. See, for example, `/etc/ssh/sshd_config`. Thus, suggest changing commented out line above to WARNING.